### DISCUSSION TITLE
Subscription option for Api and Operation scope

### DISCUSSION THREAD

**Discussion by ppandari** [Azure/api-management-developer-portal#2576] [Feature request]
New API Portal provides subscribe option at product scope. Need a way to provide subscription option for Api and Operations scopes as well

**Comment by ghost** [comment]
@ppandari, thank you for opening this issue. We will triage it within the next few business days.

**Comment by mikebudzynski** [comment]
@ppandari, currently API Management doesn't allow subscriptions on the API operation level and API-specific subscriptions can be created by service administrators.

**Comment by ghost** [comment]
@ppandari, thank you for requesting this feature.

**Comment by ppandari** [comment]
@mikebudzynski , APIM now allows creating subscriptions at API scope- (https://docs.microsoft.com/en-us/azure/api-management/api-management-subscriptions). But Developer portal do not have the option to show the Subscribe button at API scope.
